ascii/src/lib.rs

```diff
@@ -1,6 +1,52 @@
+//! # onefetch-ascii
+//!
+//! Provides the ascii template interface for [onefetch](https://github.com/
+//!
+//! ```rust
```

Comment:
```suggestion
//! ```rust,no_run
```
Since this code block is an example, not a test, I think it would be preferably to only compile it, but not run it.

Reply:
Done! https://github.com/o2sh/onefetch/pull/936/commits/00beed312def603eb99b6ba28d76459e60e1a53e